internal/commands/push.go

```diff
@@ -43,6 +44,7 @@ func pushCmd(dockerCli command.Cli) *cobra.Command {
 	}
 	flags := cmd.Flags()
 	flags.StringVarP(&opts.tag, "tag", "t", "", "Target registry reference (default: <name>:<version> from metadata)")
+	flags.StringSliceVar(&opts.platforms, "platforms", nil, "For multi-arch service images, only push the sepcified platforms")
```

Comment:
`s/platforms/platform` to match what we have in docker/cli for other `StringSliceVar`s.

Reply:
s/sepcified/specified